libpod/kube.go

```diff
@@ -957,53 +957,44 @@ func containerToV1Container(ctx context.Context, c *Container, getService bool)
 	kubeContainer.Name = removeUnderscores(c.Name())
 	_, image := c.Image()
 
-	// The infra container may have been created with an overlay root FS
```

Comment:
I still somehow feel removing this is breaks a backward compatibility - if someone specifies custom infra image based on the rootfs (if that's even possible), then with my code, we will no longer fill-in some details based on the pause image, because we do not have pause "image" anymore.

I was thinking  that maybe I could just fill-in those details using new function similar to PrepareInfraSpec, but I would like more opinions, because the tests pass even with the current code.